### PR TITLE
spawn medical tools in ambulance

### LINKED ISSUE #13364: Are stethoscope spawns too rare?
Suggested fix, new items. Chalk board, chalk, and a "just for fun" religious text titled "RnJesus, debugging and you."

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/26027509-are-stethoscope-spawns-too-rare?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by illi-kun**
including stethoscope, so it closes #13364.

### CODE REVIEW

File: data/json/vehicles/emergency.json [hunk 1]
```diff
@@ -519,17 +519,20 @@
             {"x": -2, "y": 0, "chance": 15, "items": ["aspirin"]},
             {"x": -2, "y": 0, "chance": 15, "items": ["disinfectant"]},
             {"x": -2, "y": 0, "chance": 5, "items": ["codeine"]},
+            {"x": -2, "y": 0, "chance": 25, "item_groups": "ambulance_equipment"},
             {"x": -3, "y": 0, "chance": 15, "items": ["bandages"]},
             {"x": -3, "y": 0, "chance": 15, "items": ["1st_aid"]},
-            {"x": -3, "y": 0, "chance": 10, "items": ["adrenaline_injector"]},
+            {"x": -3, "y": 0, "chance": 5, "items": ["adrenaline_injector"]},
```
Comment (illi-kun) [inline data/json/vehicles/emergency.json]: it's included in the item group, so I've nerfed chance here a bit for balance.

**Comment by chaosvolt** [comment]
Ah, good to see.

**Comment by nonymous** [comment]
Can confirm, found stethoscope in an ambulance in my current game.  I was so shocked I quit to come read the devlogs.

**Comment by chaosvolt** [comment]
Huehuehue. And now the random red mage can moonlight as a safecracker. XP

**Pull request merged**